awx_collection/test/awx/test_inventory_source.py

```diff
@@ -45,30 +46,6 @@ def test_inventory_source_create(run_module, admin_user, base_inventory, project
     }
 
 
-@pytest.mark.django_db
-def test_create_inventory_source_implied_org(run_module, admin_user):
```

Comment:
I'm removing this test because there will no longer be implied orgs, the `organization` parameter is now explicitly required for the `tower_inventory_source` module (README has been updated with this info).